packages/eslint-config-react-native/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
     },
   },
 
-  extends: ['plugin:prettier/recommended'],
+  extends: ['prettier'],
```

Comment:
https://github.com/prettier/eslint-config-prettier/blob/5687e7de69ac132f053b4bc5474d5963aa3bf6d6/README.md#L7

> Note that this config _only_ turns rules _off,_ so **it only makes sense using it together with some other config.**

```suggestion
```

Reply:
I suspect that people probably rely on it being turned off. So without this, the upgrade path might be too annoying / non-obvious. I think we can remove it separately later.

Reply:
It feels like I’m missing something. In what situation does it help to have `extends: ['prettier']` here? Remember that `rules` win over `extends`.